quickstarts/deploy.markdown

```diff
@@ -0,0 +1,59 @@
+Kroxylicious is a Java application based on [Netty](https://netty.io/), which means it will run anywhere you can run a JVM. (That's a lot of places!)
+To help you get started with Kroxylicious, we've created this quick setup guide.
+
+<br />
+
+# Getting started
+
+### Prerequisites
+
+#### Java
+
+To get started deploying Kroxylicious, you will need to install a Java Runtime Environment (JRE) with minimum version 17. This does not come included with Kroxylicious.
+
+Some operating systems come with a JRE already installed. You can check what Java version you have installed by running the following command:
+
+```shell
+java -version
+```
+
+If you get an error or the command doesn't return anything, you may not have a JRE installed, or your JRE may not have been correctly added to your system's `PATH` variable.
+
+#### Kafka
+
+You will also need a running Kafka cluster to point Kroxylicious at. The Kafka cluster address used by Kroxylicious can be changed in the configuration YAML (see the [**Configure**](#configure) section below).
+
+<br />
+
+### Downloading Kroxylicious
+
+Kroxylicious can be downloaded from the [releases](https://github.com/kroxylicious/kroxylicious/releases) page of the Kroxylicious GitHub repository, or from Maven Central.
+
+In GitHub, all releases since v0.4.0 have an attached `kroxylicious-app-*-bin.zip` file. Download this, and optionally verify the contents of the package with the attached `kroxylicious-app-*-bin.zip.asc` file.
+
+<br />
+
+# Install
+
+Extract the downloaded Kroxylicious Zip file into the directory you would like to install Kroxylicious in.
+Ensure the `kroxylicious-start.sh` and `run-java.sh` files in the `bin/` directory within the extracted folder have at least read and execute (`r-x`) permissions for the owner.
```

Comment:
Zip has maintained file permissions for a long time, so whilst harmless, I think the suggest to check file permissions is superfluous.

Reply:
There are some (probably old) versions of Zip that don't preserve permissions, and I figure it saves us having to answer this question down the line.